controllers/topics/index.js

```diff
@@ -304,7 +317,7 @@ const getFollowerList = async (req, res) => {
         END as is_following,
         ARRAY( select name from topics as tp
           left join user_topics as utp on tp.topic_id = utp.topic_id
-          where utp.user_id = "users".user_id limit 3
+          where utp.user_id = "users".user_id and tp.topic_id in (:topicIds) limit 3
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line could potentially lead to an SQL injection vulnerability because the `topicIds` array is directly interpolated into the SQL string. Always use parameter binding for variables in SQL queries to avoid SQL injection attacks.
```diff
-           where utp.user_id = "users".user_id and tp.topic_id in (:topicIds) limit 3
+           where utp.user_id = "users".user_id and tp.topic_id in (?) limit 3
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->